critiquebrainz/frontend/views/test/test_mapping.py

```diff
@@ -144,31 +149,40 @@ def test_spotify_add(self):
         self.assertIn("Test Album", str(response.data))
         self.assertIn("Test Artist", str(response.data))
 
-    def test_spotify_confirm(self):
+    @mock.patch('critiquebrainz.frontend.external.musicbrainz_db.release_group.get_release_group_by_id')
+    @mock.patch('critiquebrainz.frontend.external.spotify.get_multiple_albums')
+    @mock.patch('critiquebrainz.frontend.external.mbspotify.add_mapping')
+    @mock.patch('critiquebrainz.frontend.external.spotify.get_album')
+    @mock.patch('critiquebrainz.frontend.external.spotify.search')
```

Comment:
are all of these mocks used in each of the tests that are in this method? I think it might be more explicit if we have separate methods for each, and we only add mocks for the methods that we know we need to mock.

However, if you want to check `assert_not_called` on these mocks, perhaps this is ok.

Reply:
Yeah, I checked and removed some unneeded mocks after your review but these four are needed and and cannot be removed. We could break this into parts but at least one test requires all 4 mocks.

Reply:
Also, this is a part of the mapping code. I hope this will soon get removed once we add BrainzPlayer.